generator/templates/Authorisation/src/components/authorisation/PasswordForgottenCard.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <v-card height="100%">
+    <v-form ref="form" v-model="isValid" @submit.prevent="handleRegister()">
+      <VCardTitle class="title" v-text="$t('authorisation.passwordForgotten.title')"/>
+      <v-card-text>
+        <v-alert
+            :type="alertType"
+            :value="!!alertMessage.length"
+            class="mb-10"
+            transition="fade-transition"
+        >
+          {{ alertMessage }}
+        </v-alert>
+        <k-field-group language-prefix="authorisation.fields">
+          <KTextField
+              v-model="form.email"
+              autocomplete="username"
+              autofocus
+              field="email"
+              tabindex="1"
+          />
+        </k-field-group>
+      </v-card-text>
+      <v-card-actions>
+        <VSpacer/>
+        <VBtn
+            :to="{name:'login'}"
+            color="primary"
+            text
+            v-text="$t('actions.back')"/>
+        <VBtn :loading="isLoading"
+              color="primary"
+              tabindex="2"
+              type="submit"
+              v-text="$t('authorisation.passwordForgotten.request')"/>
+      </v-card-actions>
+    </v-form>
+  </v-card>
+</template>
```

Comment:
Je moet het wel willen - maar ik zie hier veel structuur vergeleken met components/authorisation/LoginCard.vue. Wellicht een template component maken? Sowieso kwam een dergelijke card setup ook vaak terug in S4U.